tests/installation/Jenkinsfile

```diff
@@ -0,0 +1,277 @@
+      ansible_host     : 'marist-1',
+      ssh_hostport     : 'ssh-marist-server-zzow01-hostport',
+      ssh_userpass     : 'ssh-marist-server-zzow01'
+    ],
+    'marist-2': [
+      ansible_host     : 'marist-2',
+      ssh_hostport     : 'ssh-marist-server-zzow02-hostport',
+      ssh_userpass     : 'ssh-marist-server-zzow02'
+    ],
+    'marist-3': [
+      ansible_host     : 'marist-3',
+      ssh_hostport     : 'ssh-marist-server-zzow03-hostport',
+      ssh_userpass     : 'ssh-marist-server-zzow03'
+    ],
+    'river': [
+      ansible_host     : 'river-0',
+      ssh_hostport     : 'ssh-river-original-hostport',
+      ssh_userpass     : 'ssh-zdt-test-image-guest'
+    ],
+    'river-3': [
+      ansible_host     : 'river-3',
+      ssh_hostport     : 'ssh-river-c3-hostport',
+      ssh_userpass     : 'ssh-zdt-test-image-guest'
+    ]
+  ];
+  // Test scopes, what test should we run
+  Map TEST_SCOPES = [
+    'conv-build': 'src/__tests__/basic/install.js',
+    'smpe-fmid': 'src/__tests__/basic/install-fmid.js',
+  ]
+
+  def lib = library("jenkins-library").org.zowe.jenkins_shared_library
+
+  def pipeline = lib.pipelines.generic.GenericPipeline.new(this)
+
+  pipeline.admins.add("jackjia")
+
+  // we have extra parameters for integration test
+  pipeline.addBuildParameters(
+    booleanParam(
+      name: 'STARTED_BY_AUTOMATION',
+      description: 'If this task is started with pipeline automation. Set to true if you want to skip the Continue prompt question.',
+      defaultValue: false
+    ),
+    // >>>>>>>> parameters of where and what to test
+    choice(
+      name: 'TEST_SERVER',
+      choices: TEST_SERVERS.keySet().collect(),
+      description: 'Choose which server to run test',
+      trim: true
+    ),
+    choice(
+      name: 'TEST_SCOPE',
+      choices: TEST_SCOPES.keySet().collect(),
+      description: 'Choose scope of test',
+      trim: true
+    ),
+    // >>>>>>>> parameters of artifactory
+    string(
+      name: 'ZOWE_ARTIFACTORY_PATTERN',
+      description: 'Zowe artifactory download pattern',
+      defaultValue: 'libs-snapshot-local/org/zowe/*.pax',
+      trim: true,
+      required: true
+    ),
+    string(
+      name: 'ZOWE_ARTIFACTORY_BUILD',
+      description: 'Zowe artifactory download build',
+      defaultValue: 'zowe-install-packaging :: staging',
+      trim: true
+    ),
+    string(
+      name: 'ZOWE_CLI_ARTIFACTORY_PATTERN',
+      description: 'Zowe artifactory download pattern',
+      defaultValue: 'libs-snapshot-local/org/zowe/cli/zowe-cli-package/*.zip',
+      trim: true,
+      required: true
+    ),
+    string(
+      name: 'ZOWE_CLI_ARTIFACTORY_BUILD',
+      description: 'Zowe artifactory download build',
+      defaultValue: 'Zowe CLI Bundle :: master',
+      trim: true
+    ),
+    // >>>>>>>> parametters for test cases
+    choice(
+      name: 'NODE_VERSION',
+      choices: ['v8.16.0', 'v6.17.0', 'v8.16.1', 'v8.16.2', 'v8.17.0', 'v12.13.0', 'v12.14.1'],
```

Comment:
8.16.1 is officially not supported due to issues with it (https://docs.zowe.org/stable/troubleshoot/app-framework/app-known-issues.html#desktop-apps-fail-to-load), so maybe remove it from the list?